x-pack/plugins/timelines/public/components/actions/timeline/cases/add_to_case_action.tsx

```diff
@@ -86,23 +95,40 @@ const AddToCaseActionComponent: React.FC<AddToCaseActionProps> = ({
     dispatch(setOpenAddToNewCase({ id: eventId, isOpen: false }));
   }, [dispatch, eventId]);
 
-  const createCaseFlyoutProps = useMemo(() => {
+  const createCaseFlyoutProps: GetCreateCaseFlyoutProps = useMemo(() => {
+    const { ruleId, ruleName } = normalizedEventFields(event);
+    const attachments = [
+      {
+        alertId: eventId,
+        index: eventIndex ?? '',
+        rule: {
+          id: ruleId,
+          name: ruleName,
+        },
+        owner,
+        type: CommentType.alert as const,
+      },
+    ];
     return {
-      afterCaseCreated: attachAlertToCase,
+      afterCaseCreated: onCaseCreated,
```

Comment:
I want to rename the `cases` callback `afterCaseCreated` to `onCaseCreatedSuccess`